src/scm/plams/interfaces/adfsuite/ams.py

```diff
@@ -272,6 +272,8 @@ def get_molecule(self, section: str, file: str = "ams") -> Molecule:
         sectiondict = self.read_rkf_section(section, file)
         if sectiondict:
             return Molecule._mol_from_rkf_section(sectiondict)
+        else:
+            raise ValueError(f"Could not retrieve molecule from rkf section '{section}' for file '{file}.rkf'")
```

Comment:
This is technically a small breaking change as this previously would have returned `None`, but I think it is better to raise the error